feat: add a flag to be read from yaml to choose if the start time is …

N-Plx opened the pull request:
…used or ignored

N-Plx committed:
feat: add a flag to be read from yaml to choose if the start time is used or ignored

mathieuouillon commented:
The fix works, thanks. 
Here is an event from simulation using gemc/dev
<img width="1752" height="718" alt="image" src="https://github.com/user-attachments/assets/6d9597d0-a196-4467-add0-fd9138627ba1" />

The YAML flag works fine also.

mathieuouillon reviewed (approved):
Look good for me. Work with simulation.

c-dilks committed:
ci: re-trigger

CI build job was failing due to a Maven server-side misconfiguration;
this run should succeed.
